Gemfile

```diff
@@ -1,12 +1,13 @@
 source 'https://rubygems.org'
 
-
+gem 'bootstrap-sass', '~> 3.3.7'
```

Comment:
it is not bad use gem to load bootstrap, but it is better to have assets managed by the same tool, in this case we will sooner or later user yarn for do it from the package.json. Mix assets precompile with node_modues is another fight